src/Files.App/Dialogs/SettingsDialog.xaml

```diff
@@ -116,16 +116,25 @@
 					AccessKey="F"
 					AutomationProperties.AutomationId="SettingsItemFolders"
 					Content="{helpers:ResourceString Name=FilesAndFolders}"
-					Tag="3">
+					Tag="FoldersPage">
 					<NavigationViewItem.Icon>
 						<FontIcon Glyph="&#xE8B7;" />
 					</NavigationViewItem.Icon>
 				</NavigationViewItem>
+				<NavigationViewItem
+					AccessKey="K"
+					AutomationProperties.AutomationId="SettingsItemActions"
```

Comment:
Please make sure to add this to the automation tests.

Reply:
Added to the todo. Thanks.